index.html

```diff
@@ -39,24 +39,170 @@
+				  <div class="works__card__topics flex">
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				  </div>
+				  <div class="works__card__btn">
+					  <button type="button" class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white">See Project</button>
+				  </div>
+			  </div>
+		  </li>
+		  <li class="works__card flex bg-white">
+			<div class="works__card__img">
+				<img src="Image/multi-post-project.png" alt="multi-post stories project">
+			</div>
+			<div class="works__card__text">
+				<h2 class="works__card__title fw-700 fz-32 lh-44 clr-n-800">Multi-Post Stories</h2>
+				<div class="works__card__subtitle flex">
+					<p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">CANOPY</p>
+					<ul class="works__card__subtitle-ul flex">
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					</ul>
+				</div>
+				<p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<div class="works__card__topics flex">
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				</div>
+				<div class="works__card__btn">
+					<button type="button" class="btn fw-500 fz-17 lh-24 clr-p-500 bg-white">See Project</button>
+				</div>
+			</div>
+		</li>
+		<li class="works__card flex bg-white">
+			<div class="works__card__img">
+				<img src="Image/tonic-orange.png" alt="tonic orange project">
+			</div>
+			<div class="works__card__text">
+				<h2 class="works__card__title fw-700 fz-32 lh-44 clr-n-800">Tonic</h2>
+				<div class="works__card__subtitle flex">
+					<p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">CANOPY</p>
+					<ul class="works__card__subtitle-ul flex">
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						<li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					</ul>
+				</div>
+				<p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				<div class="works__card__topics flex">
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					<p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				</div>
```

Comment:
- [ ] Same remark about technologies, the list of project technologies are a list ul and each of them a li, please kindly fix it 😉 😉